ReWeb/Server.ml

```diff
@@ -204,7 +204,7 @@ let serve ~port server =
     ~request_handler
     ~error_handler
   in
-  let listen_addr = Unix.(ADDR_INET (inet_addr_loopback, port)) in
+  let listen_addr = Unix.(ADDR_INET (inet_addr_any, port)) in
```

Comment:
@yawaramin do you want to make this configurable as well. If so, how would you like to do it?

Reply:
If the server continues to work as before, then I think no need to make it configurable.